### DISCUSSION TITLE
hashlib sha256 issues

### DISCUSSION THREAD

**Discussion by devzone777** [micropython/micropython#10094] [Libraries & Drivers]
Hi, does hashlib work on the current build (1.19.1-717-g3c1a2a942) or do I have to compile it in there myself? There is too much conflicting info about this all over the internet.

**Comment by dlech** [comment]
It depends on the port/board. It is selected by default for `MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EXTRA_FEATURES`.
